wagtail/core/models/audit_log.py

```diff
@@ -100,7 +99,7 @@ def log_action(self, instance, action, **kwargs):
                 % (instance,)
             )
 
-        data = kwargs.pop("data", "")
+        data = kwargs.pop("data", {}) or {}
```

Comment:
This is just in case the user uses `""`, `[]`, or other falsy values.

Reply:
The `kwargs.pop("data", {})`, if `data` isn't present, will return `{}` and then immediately get overwritten because `{}` is also an empty value? Nitpick, but I think `None` might be clearer here - also, I suggest adding a comment to the effect that this is to prevent `""` due to the old `TextField` type, since usually we wouldn't need to be quite so defensive.